spec/board_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/board.rb'
+
+RSpec.describe Board do
+
+  let(:board){@board = Board.new}
+
+  describe '#display' do
+    output = 1..9
+    it 'displays the board ' do
+     expect(board.display).to eql(output)
+    end   
+  end 
+
+  describe '#position_taken?' do 
+    let(:board){@board = Board.new}
+    it 'checks if there is space on the board' do
+      expect(board.position_taken?(1)).to eql(false)
+    end
+  end
+
+  describe '#win?' do
+    let(:board){Board.new} 
+    it 'checks for win on board' do
+      expect(board.win?).to be false

```

Comment:
Let's add case when there's a winner. 
  > Use your common sense: test positive and negative scenarios.

The third line in the suggestion interately adds `x` to the board three times making `x` show appear in the first vertical line of the board, this make the `win?` the method detects the winner.

You can do for other positions.

```suggestion
      expect(board.win?).to be false
      (0..2).each { |i| board.replace_var(i, 'x') }
      expect(board.win?).to be true
```

Reply:
done